libs/x-content/build.gradle

```diff
@@ -78,7 +78,7 @@ def generateProviderManifest = tasks.register("generateProviderManifest") {
   }
 }
 
-def generateProviderImpl = tasks.register("generateProviderImpl", Copy) {
+def generateProviderImpl = tasks.register("generateProviderImpl", Sync) {
```

Comment:
[This](https://docs.gradle.org/current/dsl/org.gradle.api.tasks.Sync.html) ensures the target directory is empty prior to use. (this bit me when manually testing as i could not figure out why the old version was still there)

Reply:
TIL 👍